src/locale/languages/de.js

```diff
@@ -0,0 +1,335 @@
+      preparingGitPoint: 'GitPoint vorbereiten ...',
+      cancel: 'STORNIEREN',
+      welcomeTitle: 'Willkommen bei GitPoint',
+      welcomeMessage: 'Der leistungsstärkste GitHub-Client, der zu 100% frei ist',
+      notificationsTitle: 'Kontrollbenachrichtigungen',
+      notificationsMessage:
+        'Alle ungelesenen und teilnehmenden Benachrichtigungen anzeigen und kontrollieren',
+      reposTitle: 'Repositories und Benutzer',
+      reposMessage:
+        'Einfaches Repository, Benutzer- und Organisationsinformationen',
+      issuesTitle: 'Probleme und Zuganfragen',
+      issuesMessage:
+        'In Gesprächen kommunizieren, Pull-Anfragen und mehr zusammenführen',
+      signInButton: 'ANMELDEN',
+    },
+    welcome: {
+      welcomeTitle: 'Willkommen bei GitPoint',
+    },
+    events: {
+      welcomeMessage:
+        'Herzlich willkommen! Dies ist Ihr News-Feed - Sie werden mit den neuesten Aktivitäten auf den Repositories und den Personen, denen Sie folgen, Schritt halten.',
+      commitCommentEvent: 'hat Commit kommentiert',
+      createEvent: 'erstellt {{object}}',
+      deleteEvent: 'gelöscht {{object}}',
+      issueCommentEvent: '{{action}} auf {{type}}',
+      issueEditedEvent: '{{action}} ihren Kommentar zu {{type}}',
+      issueRemovedEvent: '{{action}} ihren Kommentar zu {{type}}',
+      issuesEvent: '{{action}} Problem',
+      publicEvent: {
+        action: 'gemacht',
+        connector: 'Öffentlichkeit',
+      },
+      pullRequestEvent: '{{action}} Anforderung anziehen',
+      pullRequestReviewEvent: '{{action}} Pull-Request-Überprüfung',
+      pullRequestReviewCommentEvent: '{{action}} auf Zuganfrage',
+      pullRequestReviewEditedEvent: '{{action}} ihr Kommentar zur Pull-Anfrage',
+      pullRequestReviewDeletedEvent: '{{action}} ihr Kommentar zur Pull-Anfrage',
+      releaseEvent: '{{action}} Freisetzung',
+      atConnector: 'beim',
+      toConnector: 'beim',
+      types: {
+        pullRequest: 'Anforderung anziehen',
+        issue: 'Problem',
+      },
+      objects: {
+        repository: 'Repository',
+        branch: 'Ast',
+        tag: 'tag',
+      },
+      actions: {
+        added: 'hinzugefügt',
+        created: 'erstellt',
+        edited: 'bearbeitet',
+        deleted: 'gelöscht',
+        assigned: 'zugewiesen',
+        unassigned: 'nicht zugeordnet',
+        labeled: 'gekennzeichnet',
+        unlabeled: 'unbeschriftet',
+        opened: 'geöffnet',
+        milestoned: 'Meilenstein',
+        demilestoned: 'demilestoned',
+        closed: 'abgeschlossen',
+        reopened: 'wiedereröffnet',
+        review_requested: 'Überprüfung angefordert',
+        review_request_removed: 'Überprüfungsanforderung entfernt',
+        submitted: 'eingereicht',
+        dismissed: 'entlassen',
+        published: 'veröffentlicht',
+        publicized: 'publiziert',
+        privatized: 'privatisiert',
+        starred: 'Stern',
```

Comment:
Is this exactly true? "starred" a verb, but `Stern` a noun.

Reply:
Yaa, i'll change it to starred